### ISSUE TITLE
Feature Request: Pie Chart Generation

### ISSUE THREAD

**Issue by RiccLQL** [excalidraw/excalidraw#4255]
I'm absolutely amazed by the copy/paste charting feature, and my girlfriend uses it to generate quick charts for presentations. I wanted to help her by creating the option to also generate pie charts.

I can implement it but based on past discussions it looks like there are several challenges (i.e. unique colors). I was wondering if anyone already has tips, ideas and ways to build these pie charts - I think it would be a useful chart option to add:)

**tl:dr: Can we implement pie chart generation? I can do it.**
